### PR TITLE
Spanish terminology

### PR DESCRIPTION
**Pull request by BlackLotus64**
Resolves one of the five in #5.

@lh1008 @erciccione

### CODE REVIEW

File: spanish-terminology.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+| Spend Key | Clave de utilización | |
+| Stagenet | Fase de la red | Para discusión. Dejarla como está o traducirla |
+| Tail emission | Emisión final | A discusión porque emisión de "cola" o "trasera" puede mal entenderse |
+| Untraceable | No rastreable | Para slogan se propone "Imposible de rastrear", para lo demás, "no rastreable" |
+| View Key | Clave de visualización | |
+| Wallet | Monedero | Se tomó "monedero" porque ya está siendo utilizado en la comunidad de Bitcoin |
+| White paper | Libro blanco | Wikipedia define el término como Libro blanco |
+
+
+## Not translated
```
Comment (erciccione) [inline spanish-terminology.md]: This section does not display properly for some reason

Comment (BlackLotus64) [reply spanish-terminology.md]: It works now, missed a line for the table template.
BTW how do I quote a specific line?

**Comment by lh1008** [comment]
BTW is great to have the terminology worked over here. I have to get started with it. How is can I add more words @BlackLotus64? Fork monero-translations repository and update the file?

**Comment by erciccione** [comment]
@lh1008 better open another PR to add more words, that to keep the history clean, merging this now.

**Pull request merged**